frontend/src/app/shared/scss/_mixins.scss

```diff
@@ -5,3 +5,27 @@
   -moz-user-select: none;
   -ms-user-select: none;
 }
+
+@mixin default-poppins-text {
+  font-family: 'Poppins';
+  font-style: normal;
+  font-weight: 600;
+  font-size: 12px;
+  line-height: 18px;
+  color: $color-grey;
+}
+
+@mixin default-open-sans-text {
+  font-family: 'Open Sans';
+  font-style: normal;
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 19px;
+  color: $color-grey;
+}
+
+@mixin reset-list {
+  margin: 0;
+  padding: 0;
+  list-style: none;
+}
```

Comment:
cred ca toate ul-urile din viata noastra ar trebui resetate.... gen nu cred ca avem nevoie de acest mixin.

mai bine pui in _layouts.scss ca 
```

ul {
  margin: 0;
  padding: 0;
  list-style: none;
}
```